src/app/categories/[slug]/page.tsx

```diff
@@ -1,4 +1,3 @@
-'use client';
```

Comment:
# Reviewer Note
This was the problematic statement causing dynamic page generation.

Reply:
That's what I also thought at first.